client/cli/src/params/pruning_params.rs

```diff
@@ -30,11 +30,16 @@ pub struct PruningParams {
 	/// 256 blocks.
 	#[structopt(long = "pruning", value_name = "PRUNING_MODE")]
 	pub pruning: Option<String>,
+	/// Specify the number of finalized blocks to keep in the database.
+	///
+	/// Default is to keep all blocks.
+	#[structopt(long = "block-pruning", value_name = "COUNT")]
+	pub block_pruning: Option<u32>,
```

Comment:
Should we maybe also go for `keep_blocks` here to make it consistent over the code base?